docs/reference/operations/create_accounts.md

```diff
@@ -0,0 +1,112 @@
+
+## Result
+
+Results are listed in this section in order of descending precedence — that is, if more than one
+error is applicable, only the one listed first is returned.
+
+### `ok`
+The account was successfully created, but did not previously exist.
+
+Note that `ok` is generated by the client implementation; the network protocol does not include a
+result when the account was successfully created.
+
+### `linked_event_failed`
+The account was not created.
+One or more of the accounts in the [linked chain](../accounts.md#flagslinked) is invalid, so the
+whole chain failed.
+
+### `linked_event_chain_open`
+The account was not created.
+The [`Account.flags.linked`](../accounts.md#flagslinked) flag was set on the last event in the
+batch, which is not legal. (`flags.linked` indicates that the chain continues to the next
+operation).
+
+### `reserved_flag`
+The account was not created.
+[`Account.flags.reserved`](../accounts.md#flagsreserved) is nonzero, but must be zero.
+
+### `reserved_field`
+The account was not created.
+[`Account.reserved`](../accounts.md#reserved) is nonzero, but must be zero.
+
+### `id_must_not_be_zero`
+The account was not created.
+[`Account.id`](../accounts.md#id) is zero, which is a reserved value.
+
+### `id_must_not_be_int_max`
+The account was not created.
+[`Account.id`](../accounts.md#id) is `2^128 - 1`, which is a reserved value.
+
+### `ledger_must_not_be_zero`
+The account was not created.
+[`Account.ledger`](../accounts.md#ledger) is zero, but must be nonzero.
+
+### `code_must_not_be_zero`
+The account was not created.
+[`Account.code`](../accounts.md#code) is zero, but must be nonzero.
+
+### `debits_pending_must_be_zero`
+The account was not created.
+[`Account.debits_pending`](../accounts.md#debits_pending) is nonzero, but must be zero.
+
+An account's debits and credits are only modified by transfers.
+
+### `debits_posted_must_be_zero`
+The account was not created.
+[`Account.debits_posted`](../accounts.md#debits_posted) is nonzero, but must be zero.
+
+An account's debits and credits are only modified by transfers.
+
+### `credits_pending_must_be_zero`
+The account was not created.
+[`Account.credits_pending`](../accounts.md#credits_pending) is nonzero, but must be zero.
+
+An account's debits and credits are only modified by transfers.
+
+### `credits_posted_must_be_zero`
+The account was not created.
+[`Account.credits_posted`](../accounts.md#credits_posted) is nonzero, but must be zero.
+
+An account's debits and credits are only modified by transfers.
+
+### `mutually_exclusive_flags`
+The account was not created.
+An account cannot be created with the specified combination of
+[`Account.flags`](../accounts.md#flags).
+
+The following flags are mutually exclusive:
+- [`Account.flags.debits_must_not_exceed_credits`](../accounts.md#flagsdebits_must_not_exceed_credits)
+- [`Account.flags.credits_must_not_exceed_debits`](../accounts.md#flagscredits_must_not_exceed_debits)
```

Comment:
Dumb question: why are these mutually exclusive? It's not possible you'd want to make sure an account is completely balanced? Maybe that's nonsense. Just wondering

Reply:
If both `exceed` flags were set, you would never be able to change its balance.

Reply:
You could, as part of a transaction (which I guess are linked events), enforce that an account always gets transferred some amount when it transfers some amount out.

Reply:
Well, not with how the state machine works right now :wink:.
In particular, that would change the state machine semantics, which currently are simple because each transfer is executed serially, and its constraints are valid at the end of every transfer. For "equal-in-and-out" to work, transfers must logically execute simultaneously, which is harder to reason about.